source/includes/stackpath/_network_policy_rules.md

```diff
@@ -0,0 +1,44 @@
+## Network policy rules
+
+Network policy rules allows you to control inbound and outbound traffic to your [workload](#stackpath-workloads).
+
+
+<!-------------------- CREATE A NETWORK POLICY RULE -------------------->
+
+### Create a network policy rule
+
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+  -d "request_body" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules?workloadId=bf9fd2ac-f761-46ef-88e0-b61ef68f8619"
```

Comment:
Isn'it supposed to have a backslash like so:
`https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules/?workloadId=bf9fd2ac-f761-46ef-88e0-b61ef68f8619`

As per line 29?

Reply:
networkpolicyrules/?workloadId
or
networkpolicyrules?workloadId

the latter makes more sense but both are present in the docs.

Reply:
Yes, it should be `networkpolicyrules?workloadId`.
Fixed in 3f79950